common.sh

```diff
@@ -384,6 +384,12 @@ export AGENT_E2E_TEST_BOOT_MODE=${AGENT_E2E_TEST_BOOT_MODE:-"ISO"}
 # to open a firewall port
 export AGENT_PXE_SERVER_PORT=${AGENT_PXE_SERVER_PORT:-8089}
 
+# Ports that need to be open for platform "none". Hosts inside the libvirt firewalld zone
+# need to communicate with haproxy deployed on the libvirt host.
+export AGENT_MCS_PORT=22623
```

Comment:
nit: not sure about the word `MCS`, I'd prefer a more speaking one

Reply:
Going to remove these environment variable as the port numbers cannot be changed.